src/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { Text } from 'react-native';
+
+export class ErrorBoundary extends React.Component<{}, { hasError: boolean }> {
+    public static getDerivedStateFromError(error) {
+        // Update state so the next render will show the fallback UI.
+        return { hasError: true };
+    }
+
+    public state = { hasError: false };
+
+    public componentDidCatch(error, info) {
+        console.log(error, info);
+    }
+
+    public render() {
+        if (this.state.hasError) {
+            return <Text>Something went wrong.</Text>;
```

Comment:
It's better to not show error to the user when there are no meaningful action that they can do, so I would just return with null here (and log the error of course)

Reply:
I have a hunch this is not true, let's compare the 2 screenshots.

Reply:
What I'm saying is that when you return this text, the user will see a post with the text 'Something went wrong'. From a user perspective this is not so good, because this is not the original text of the post, but some users may think that is. Other users may think that there was an error with this post, but since there are no actions that they can do about it, it's not so useful.
I think the best would be to hide posts that has errors in it (as well as not let users post content that we cannot display later), but I don't know how to do it at this stage.

Reply:
Also depends on what we mean by logging. If we don't want automatic / opt-in logging with some service, it might be useful to give the users (maybe only in the beta or a tester user group) some kind of error message with an actionable item - sending the logs. I've opened an issue for it https://github.com/agazso/postmodern/issues/103